### PR TITLE
Add Skip import option to system props rule

### PR DESCRIPTION
**Pull request by SferaDev**
- Add skip import check
- Auto-fix lint warnings
- Add exclusions to ``CircleOcticon`` and ``Token``

### CODE REVIEW

File: docs/rules/no-system-props.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ import {Avatar} from 'some-other-library'
 
 ## Options
 
+- `skipImportCheck` (default: `false`)
+
+  By default, the `no-system-props` rule will only check for styled system props used in functions and components that are imported from `@primer/components`. You can disable this behavior by setting `skipImportCheck` to `true`. This is useful for linting custom components that pass styled system props down to Primer React components.
```
Comment (colebemis) [inline docs/rules/no-system-props.md:46]:
```suggestion
  By default, the `no-system-props` rule will only check for styled system props used in functions and components that are imported from `@primer/react`. You can disable this behavior by setting `skipImportCheck` to `true`. This is useful for linting custom components that pass styled system props down to Primer React components.
```

**Review by colebemis — APPROVED** [review]
Thanks, @SferaDev ❤️

**Comment by colebemis** [comment]
@SferaDev Looks like there are some CI failures. Happy to merge this when those are fixed ✅

File: src/rules/no-system-props.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ const utilityComponents = new Set(['Box', 'Text'])
 const excludedComponentProps = new Map([
   ['AnchoredOverlay', new Set(['width', 'height'])],
   ['Avatar', new Set(['size'])],
+  ['CircleOcticon', new Set(['size'])]
```
Comment (SferaDev) [inline src/rules/no-system-props.js:17]:
```suggestion
  ['CircleOcticon', new Set(['size'])],
```

**Pull request merged**